src/norm_layer_cuda.cu

```diff
@@ -789,14 +787,16 @@ layer is a convolutional layer.
         float tmp_mu_w = mu_w[div_idx];
```

Comment:
`tmp_mu_a_2` at line 786 should be removed because it is unused

Reply:
Done.